pkg/apis/kops/kubeletconfig.go

```diff
@@ -56,5 +61,27 @@ func BuildKubeletConfigSpec(cluster *Cluster, instanceGroup *InstanceGroup) (*Ku
 		c.NodeLabels[k] = v
 	}
 
+	sv, err := util.ParseKubernetesVersion(cluster.Spec.KubernetesVersion)
+	if err != nil {
+		return c, fmt.Errorf("Failed to lookup kubernetes version: %v", err)
+	}
+
+	// --register-with-taints was available in the first 1.6.0 alpha, no need to rely on semver's pre/build ordering
+	sv.Pre = nil
+	sv.Build = nil
+	if sv.GTE(semver.Version{Major: 1, Minor: 6, Patch: 0, Pre: nil, Build: nil}) {
+		for i, t := range instanceGroup.Spec.Taints {
+			if c.Taints == nil {
+				c.Taints = make([]string, len(instanceGroup.Spec.Taints))
+			}
+			c.Taints[i] = t
+		}
+
+		// Enable scheduling since it can be controlled via taints.
+		// For pre-1.6.0 clusters, this is handled by taints.go
+		registerSchedulable := true
+		c.RegisterSchedulable = &registerSchedulable
+	}
```

Comment:
I think we need an `else` block here, because user-specified taints are not supported with 1.5 AFAICT.  (We could do it, but it would be a lot of work and I don't think it is worth it).

We should just `return fmt.Errorf("User-specified taints are not supported before kubernetes version 1.6.0")` in the else case

Ah... but then that causes a conflict with the taints for the master being explicitly specified, which I do like.

It's a bit nasty, but how about we get around that by comparing the taint to the exact taints we expect for 1.5, that we know protokube will apply?